.github/workflows/test.yml

```diff
@@ -7,12 +7,33 @@ jobs:
   test:
     environment: test
     runs-on: ubuntu-20.04
+
+    services:
+      s3:
+        image: localstack/localstack
```

Comment:
Could you switch the image to `localstack/localstack:s3-latest`? The image is much smaller than the standard one.